Lib/test/test_mmap.py

```diff
@@ -1165,6 +1165,36 @@ def test_flush_parameters(self):
             m.flush(PAGESIZE)
             m.flush(PAGESIZE, PAGESIZE)
 
+    @unittest.skipUnless(sys.platform == 'linux', 'Linux only')
+    @support.requires_linux_version(5, 17, 0)
+    def test_set_name(self):
+        # Test setting name on anonymous mmap
+        m = mmap.mmap(-1, PAGESIZE)
+        self.addCleanup(m.close)
+        result = m.set_name('test_mapping')
+        self.assertIsNone(result)
+
+        # Test name length limit (80 chars including prefix "cpython:mmap:")
+        # Prefix is 13 chars, so max name is 67 chars
+        long_name = 'x' * 30
```

Comment:
I suggest to test the longest valid name here:

```suggestion
        long_name = 'x' * 67
```

And check for overflow below (68 characters).